README.md

```diff
@@ -10,11 +10,24 @@
 
 [![Cerner OSS](https://badgen.net/badge/Cerner/OSS/blue)](http://engineering.cerner.com/2014/01/cerner-and-open-source/)
 [![License](https://badgen.net/github/license/cerner/terra-graphs)](https://github.com/cerner/terra-graphs/blob/main/LICENSE)
-[![Build Status](https://badgen.net/travis/cerner/terra-graphs)](https://travis-ci.com/cerner/terra-graphs)
-[![devDependencies status](https://badgen.net/david/dev/cerner/terra-graphs)](https://david-dm.org/cerner/terra-graphs?type=dev)
+[![Build Status](https://travis-ci.com/cerner/terra-graphs.svg?branch=main)](https://travis-ci.com/cerner/terra-graphs)
 [![lerna](https://badgen.net/badge/maintained%20with/lerna/cc00ff)](https://lerna.js.org/)
+[![npm latest version](https://img.shields.io/npm/v/@cerner/carbon-graphs/latest.svg)](https://www.npmjs.com/package/@cerner/carbon-graphs)
+[![dependencies Status](https://david-dm.org/cerner/terra-graphs/status.svg)](https://david-dm.org/cerner/terra-graphs)
+[![devDependencies status](https://badgen.net/david/dev/cerner/terra-graphs)](https://david-dm.org/cerner/terra-graphs?type=dev)
+[![Bugs](https://img.shields.io/github/issues/cerner/carbon-graphs/bug.svg)](https://github.com/cerner/terra-graphs/issues?utf8=✓&q=is%3Aissue+is%3Aopen+label%3Abug)
+
+Terra Graphs is a graphing library built using D3 visualization library. It provides an API for generating native graphs such as Line and Paired Result graphs based on design standards.
 
 - [Supported Browsers](#supported-browsers)
+- [Getting Started](#getting-started)
+  - [Dependencies](#dependencies)
+  - [Install](#install)
+- [Features](#features)
+    -   [Graph types](#graph-types)
+    -   [Graph Settings/Toggles](#graph-settingstoggles)
```

Comment:
Looks like it is not pointing to the appropriate section in dev site when clicked.

Reply:
it seems to be working fine here: https://github.com/cerner/terra-graphs/blob/da9a5e020b9d714181765911e5e3e046bc83d7aa/README.md#graph-settingstoggles

Reply:
it is not working in dev site